### ISSUE TITLE
Wrong type for state.reported.connected 

### ISSUE THREAD

**Issue by thomas-ek** [home-assistant-HomeWhiz/home-assistant-HomeWhiz#261]
Hi guys, I am unable to get any information about my washing drying machine from Beko
Here are the logs from when I try to reload the device on the Home Whiz integration : 

```
024-12-18 00:07:29.949 INFO (MainThread) [custom_components.homewhiz] Setting up entry F999949956871667140013                                                                                                                                                                          │
2024-12-18 00:07:29.949 INFO (MainThread) [custom_components.homewhiz] Setting up cloud connection                                                                                                                                                                                      │
2024-12-18 00:07:29.956 INFO (MainThread) [homeassistant.components.select] Setting up homewhiz.select                                                                                                                                                                                  │
2024-12-18 00:07:29.985 DEBUG (MainThread) [custom_components.homewhiz] Selects: ['state', 'washer_program', 'washer_temperature', 'washer_spin', 'washer_dry', 'custom_duration_level', 'washer_rinse_count', 'settings_volume']                                                       │
2024-12-18 00:07:29.986 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:29.986 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:29.987 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:29.987 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:29.987 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:29.988 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:29.988 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:29.988 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:29.988 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:29.989 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:29.989 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:29.989 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:29.990 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:29.990 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:29.990 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:29.996 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:29.996 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:29.997 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:29.997 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:29.997 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:29.998 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:29.998 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:29.998 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:29.998 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:29.999 INFO (MainThread) [homeassistant.components.sensor] Setting up homewhiz.sensor                                                                                                                                                                                  │
2024-12-18 00:07:30.031 DEBUG (MainThread) [custom_components.homewhiz] Generated controls: [<custom_components.homewhiz.appliance_controls.WriteEnumControl object at 0x7fc03162c8a0>, <custom_components.homewhiz.appliance_controls.WriteEnumControl object at 0x7fc03162c0c0>, <cust│
om_components.homewhiz.appliance_controls.EnumControl object at 0x7fc032a00cd0>, <custom_components.homewhiz.appliance_controls.WriteEnumControl object at 0x7fc03162c9f0>, <custom_components.homewhiz.appliance_controls.WriteEnumControl object at 0x7fc03162ca60>, WriteBooleanContr│
ol(), WriteBooleanControl(), WriteBooleanControl(), WriteBooleanControl(), WriteBooleanControl(), WriteBooleanControl(), <custom_components.homewhiz.appliance_controls.WriteEnumControl object at 0x7fc03358eeb0>, <custom_components.homewhiz.appliance_controls.WriteEnumControl obje│
ct at 0x7fc03162d010>, <custom_components.homewhiz.appliance_controls.WriteNumericControl object at 0x7fc032a00e10>, WriteBooleanControl(), WriteBooleanControl(), WriteBooleanControl(), WriteBooleanControl(), <custom_components.homewhiz.appliance_controls.TimeControl object at 0x│
7fc0335779b0>, <custom_components.homewhiz.appliance_controls.TimeControl object at 0x7fc0315a8c00>, <custom_components.homewhiz.appliance_controls.TimeControl object at 0x7fc032a5a350>, <custom_components.homewhiz.appliance_controls.SummedTimestampControl object at 0x7fc0334f219│
0>, <custom_components.homewhiz.appliance_controls.SummedTimestampControl object at 0x7fc032a0d490>, <custom_components.homewhiz.appliance_controls.BooleanCompareControl object at 0x7fc032a00f50>, <custom_components.homewhiz.appliance_controls.BooleanBitmaskControl object at 0x7f│
c0315a8af0>, <custom_components.homewhiz.appliance_controls.BooleanBitmaskControl object at 0x7fc0315a8e20>, <custom_components.homewhiz.appliance_controls.BooleanBitmaskControl object at 0x7fc032a5a550>, <custom_components.homewhiz.appliance_controls.WriteEnumControl object at 0│
x7fc03162d470>]                                                                                                                                                                                                                                                                         │
2024-12-18 00:07:30.031 DEBUG (MainThread) [custom_components.homewhiz] Sensors: ['state', 'washer_program', 'sub_state', 'washer_temperature', 'washer_spin', 'washer_dry', 'custom_duration_level', 'washer_rinse_count', 'delay_start#0', 'washer_duration', 'washer_remaining', 'was│
her_delay', 'delay_start_time#0', 'settings_volume']                                                                                                                                                                                                                                    │
2024-12-18 00:07:30.032 DEBUG (MainThread) [custom_components.homewhiz] Entities: {'state': <entity unknown.unknown=unknown>, 'washer_program': <entity unknown.unknown=unknown>, 'sub_state': <entity unknown.unknown=unknown>, 'washer_temperature': <entity unknown.unknown=unknown>,│
 'washer_spin': <entity unknown.unknown=unknown>, 'washer_dry': <entity unknown.unknown=unknown>, 'custom_duration_level': <entity unknown.unknown=unknown>, 'washer_rinse_count': <entity unknown.unknown=unknown>, 'delay_start#0': <entity unknown.unknown=unknown>, 'washer_duration│
': <entity unknown.unknown=unknown>, 'washer_remaining': <entity unknown.unknown=unknown>, 'washer_delay': <entity unknown.unknown=unknown>, 'delay_start_time#0': <entity unknown.unknown=unknown>, 'settings_volume': <entity unknown.unknown=unknown>}                               │
2024-12-18 00:07:30.032 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:30.032 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:30.032 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:30.032 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:30.033 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key state                                                                                                                                                                                │
2024-12-18 00:07:30.033 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:30.034 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:30.034 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:30.034 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program                                                                                                                                                                       │
2024-12-18 00:07:30.034 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_program  
2024-12-18 00:07:30.034 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key sub_state                                                                                                                                                                            │
2024-12-18 00:07:30.034 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key sub_state                                                                                                                                                                            │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key sub_state                                                                                                                                                                            │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key sub_state                                                                                                                                                                            │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key sub_state                                                                                                                                                                            │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:30.035 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:30.036 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_temperature                                                                                                                                                                   │
2024-12-18 00:07:30.036 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:30.036 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:30.036 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:30.036 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:30.036 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_spin                                                                                                                                                                          │
2024-12-18 00:07:30.037 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:30.037 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:30.037 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:30.037 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:30.037 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_dry                                                                                                                                                                           │
2024-12-18 00:07:30.037 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:30.038 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:30.038 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:30.038 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:30.038 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key custom_duration_level                                                                                                                                                                │
2024-12-18 00:07:30.038 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:30.039 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:30.039 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:30.039 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:30.039 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_rinse_count                                                                                                                                                                   │
2024-12-18 00:07:30.040 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start#0                                                                                                                                                                        │
2024-12-18 00:07:30.040 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start#0                                                                                                                                                                        │
2024-12-18 00:07:30.040 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start#0                                                                                                                                                                        │
2024-12-18 00:07:30.040 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start#0                                                                                                                                                                        │
2024-12-18 00:07:30.040 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start#0                                                                                                                                                                        │
2024-12-18 00:07:30.040 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_duration                                                                                                                                                                      │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_duration                                                                                                                                                                      │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_duration                                                                                                                                                                      │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_duration                                                                                                                                                                      │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_duration                                                                                                                                                                      │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_remaining                                                                                                                                                                     │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_remaining                                                                                                                                                                     │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_remaining                                                                                                                                                                     │
2024-12-18 00:07:30.041 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_remaining                                                                                                                                                                     │
2024-12-18 00:07:30.042 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_remaining
2024-12-18 00:07:30.042 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_delay                                                                                                                                                                         │
2024-12-18 00:07:30.042 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_delay                                                                                                                                                                         │
2024-12-18 00:07:30.042 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_delay                                                                                                                                                                         │
2024-12-18 00:07:30.042 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_delay                                                                                                                                                                         │
2024-12-18 00:07:30.042 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_delay                                                                                                                                                                         │
2024-12-18 00:07:30.043 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start_time#0                                                                                                                                                                   │
2024-12-18 00:07:30.043 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start_time#0                                                                                                                                                                   │
2024-12-18 00:07:30.043 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start_time#0                                                                                                                                                                   │
2024-12-18 00:07:30.043 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start_time#0                                                                                                                                                                   │
2024-12-18 00:07:30.043 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key delay_start_time#0                                                                                                                                                                   │
2024-12-18 00:07:30.043 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:30.044 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:30.044 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:30.044 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:30.044 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key settings_volume                                                                                                                                                                      │
2024-12-18 00:07:30.045 INFO (MainThread) [homeassistant.components.climate] Setting up homewhiz.climate                                                                                                                                                                                │
2024-12-18 00:07:30.067 DEBUG (MainThread) [custom_components.homewhiz] ACs: []                                                                                                                                                                                                         │
2024-12-18 00:07:30.068 INFO (MainThread) [homeassistant.components.switch] Setting up homewhiz.switch                                                                                                                                                                                  │
2024-12-18 00:07:30.087 DEBUG (MainThread) [custom_components.homewhiz] Switches: ['washer_prewash', 'washer_extrarinse', 'washer_fast_plus', 'washer_fast', 'washer_hidden_anti_crease', 'washer_steam', 'washer_add_water', 'washer_soaking', 'washer_night', 'washer_anticrease']    │
2024-12-18 00:07:30.087 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_prewash                                                                                                                                                                       │
2024-12-18 00:07:30.087 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_prewash                                                                                                                                                                       │
2024-12-18 00:07:30.087 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_prewash                                                                                                                                                                       │
2024-12-18 00:07:30.088 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_extrarinse                                                                                                                                                                    │
2024-12-18 00:07:30.088 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_extrarinse                                                                                                                                                                    │
2024-12-18 00:07:30.088 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_extrarinse                                                                                                                                                                    │
2024-12-18 00:07:30.088 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_fast_plus                                                                                                                                                                     │
2024-12-18 00:07:30.088 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_fast_plus                                                                                                                                                                     │
2024-12-18 00:07:30.088 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_fast_plus                                                                                                                                                                     │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_fast                                                                                                                                                                          │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_fast                                                                                                                                                                          │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_fast                                                                                                                                                                          │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_hidden_anti_crease                                                                                                                                                            │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_hidden_anti_crease                                                                                                                                                            │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_hidden_anti_crease                                                                                                                                                            │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_steam                                                                                                                                                                         │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_steam                                                                                                                                                                         │
2024-12-18 00:07:30.089 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_steam                                                                                                                                                                         │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_add_water                                                                                                                                                                     │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_add_water                                                                                                                                                                     │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_add_water                                                                                                                                                                     │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_soaking                                                                                                                                                                       │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_soaking                                                                                                                                                                       │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_soaking                                                                                                                                                                       │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_night                                                                                                                                                                         │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_night                                                                                                                                                                         │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_night                                                                                                                                                                         │
2024-12-18 00:07:30.090 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_anticrease                                                                                                                                                                    │
2024-12-18 00:07:30.091 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_anticrease                                                                                                                                                                    │
2024-12-18 00:07:30.091 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_anticrease                                                                                                                                                                    │
2024-12-18 00:07:30.091 INFO (MainThread) [homeassistant.components.binary_sensor] Setting up homewhiz.binary_sensor                                                                                                                                                                    │
2024-12-18 00:07:30.101 DEBUG (MainThread) [custom_components.homewhiz] Binary sensors: ['remote_control', 'washer_warning_door_is_open', 'washer_warning_no_water', 'washer_warning_security']                                                                                         │
2024-12-18 00:07:30.101 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key remote_control                                                                                                                                                                       │
2024-12-18 00:07:30.101 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key remote_control                                                                                                                                                                       │
2024-12-18 00:07:30.101 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key remote_control                                                                                                                                                                       │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_door_is_open                                                                                                                                                          │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_door_is_open                                                                                                                                                          │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_door_is_open
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_no_water                                                                                                                                                              │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_no_water                                                                                                                                                              │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_no_water                                                                                                                                                              │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_security                                                                                                                                                              │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_security                                                                                                                                                              │
2024-12-18 00:07:30.102 DEBUG (MainThread) [custom_components.homewhiz] Retrieving translation_key washer_warning_security                                                                                                                                                              │
2024-12-18 00:07:30.103 INFO (MainThread) [custom_components.homewhiz] Connecting to F999949956871667140013                                                                                                                                                                             │
2024-12-18 00:07:30.105 INFO (MainThread) [custom_components.homewhiz] Setup cloud connection successfully                                                                                                                                                                              │
2024-12-18 00:07:30.620 DEBUG (MainThread) [custom_components.homewhiz] Credentials expire at: 2024-12-18 00:07:30+00:00                                                                                                                                                                │
2024-12-18 00:07:30.877 DEBUG (MainThread) [custom_components.homewhiz] MQTT connection successful                                                                                                                                                                                      │
2024-12-18 00:07:30.980 DEBUG (MainThread) [custom_components.homewhiz] Subscribe to update result: {'packet_id': 1, 'topic': '$aws/things/F999949956871667140013/shadow/update/accepted', 'qos': <QoS.AT_LEAST_ONCE: 1>}                                                               │
2024-12-18 00:07:30.981 DEBUG (MainThread) [custom_components.homewhiz] Subscribe to get result: {'packet_id': 2, 'topic': '$aws/things/F999949956871667140013/shadow/get/accepted', 'qos': <QoS.AT_LEAST_ONCE: 1>}                                                                     │
2024-12-18 00:07:30.981 DEBUG (MainThread) [custom_components.homewhiz] Forcing read                                                                                                                                                                                                    │
2024-12-18 00:07:30.982 DEBUG (MainThread) [custom_components.homewhiz] Force read result: {'packet_id': 3}                                                                                                                                                                             │
2024-12-18 00:07:30.984 DEBUG (MainThread) [custom_components.homewhiz] Get shadow result: {'packet_id': 4}                                                                                                                                                                             │
2024-12-18 00:07:30.984 DEBUG (MainThread) [custom_components.homewhiz] Set hass time interval update                                                                                                                                                                                   │
Exception ignored in: <class 'dacite.exceptions.WrongTypeError'>                                                                                                                                                                                                                        │
Traceback (most recent call last):                                                                                                                                                                                                                                                      │
2024-12-18 00:07:31.047 DEBUG (Dummy-37) [custom_components.homewhiz] Handling notify                                                                                                                                                                                                   │
  File "/usr/local/lib/python3.13/site-packages/awscrt/mqtt.py", line 642, in callback_wrapper                                                                                                                                                                                          │
    callback(topic=topic, payload=payload, dup=dup, qos=QoS(qos), retain=retain)                                                                                                                                                                                                        │
  File "/config/custom_components/homewhiz/cloud.py", line 145, in <lambda>                                                                                                                                                                                                             │
    lambda topic, payload, dup, qos, retain, **kwargs: self.handle_notify(                                                                                                                                                                                                              │
  File "/config/custom_components/homewhiz/cloud.py", line 240, in handle_notify                                                                                                                                                                                                        │
    message = from_dict(MqttPayload, json.loads(payload))                                                                                                                                                                                                                               │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 64, in from_dict                                                                                                                                                                                                  │
    value = _build_value(type_=field_type, data=field_data, config=config)                                                                                                                                                                                                              │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 99, in _build_value                                                                                                                                                                                               │
    data = from_dict(data_class=type_, data=data, config=config)                                                                                                                                                                                                                        │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 64, in from_dict                                                                                                                                                                                                  │
    value = _build_value(type_=field_type, data=field_data, config=config)                                                                                                                                                                                                              │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 99, in _build_value                                                                                                                                                                                               │
    data = from_dict(data_class=type_, data=data, config=config)                                                                                                                                                                                                                        │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 69, in from_dict                                                                                                                                                                                                  │
    raise WrongTypeError(field_path=field.name, field_type=field_type, value=value)                                                                                                                                                                                                     │
dacite.exceptions.WrongTypeError: wrong value type for field "state.reported.connected" - should be "bool | None" instead of value "true" of type "str"                                                                                                                                 │
Exception ignored in: <class 'dacite.exceptions.WrongTypeError'>                                                                                                                                                                                                                        │
Traceback (most recent call last):                                                                                                                                                                                                                                                      │
  File "/usr/local/lib/python3.13/site-packages/awscrt/mqtt.py", line 642, in callback_wrapper                                                                                                                                                                                          │
2024-12-18 00:07:31.105 DEBUG (Dummy-38) [custom_components.homewhiz] Handling notify                                                                                                                                                                                                   │
    callback(topic=topic, payload=payload, dup=dup, qos=QoS(qos), retain=retain)                                                                                                                                                                                                        │
  File "/config/custom_components/homewhiz/cloud.py", line 145, in <lambda>                                                                                                                                                                                                             │
    lambda topic, payload, dup, qos, retain, **kwargs: self.handle_notify(                                                                                                                                                                                                              │
  File "/config/custom_components/homewhiz/cloud.py", line 240, in handle_notify                                                                                                                                                                                                        │
    message = from_dict(MqttPayload, json.loads(payload))                                                                                                                                                                                                                               │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 64, in from_dict                                                                                                                                                                                                  │
    value = _build_value(type_=field_type, data=field_data, config=config)                                                                                                                                                                                                              │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 99, in _build_value                                                                                                                                                                                               │
    data = from_dict(data_class=type_, data=data, config=config)                                                                                                                                                                                                                        │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 64, in from_dict                                                                                                                                                                                                  │
    value = _build_value(type_=field_type, data=field_data, config=config)                                                                                                                                                                                                              │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 99, in _build_value                                                                                                                                                                                               │
    data = from_dict(data_class=type_, data=data, config=config)                                                                                                                                                                                                                        │
  File "/usr/local/lib/python3.13/site-packages/dacite/core.py", line 69, in from_dict                                                                                                                                                                                                  │
    raise WrongTypeError(field_path=field.name, field_type=field_type, value=value)                                                                                                                                                                                                     │
dacite.exceptions.WrongTypeError: wrong value type for field "state.reported.connected" - should be "bool | None" instead of value "true" of type "str"

```

**Comment by townsmcp** [comment]
You need to add more info, such as version of HA installed, model of washer dryer, if it works fine from the HomeWhiz app etc

**Comment by thomas-ek** [comment]
> You need to add more info, such as version of HA installed, model of washer dryer, if it works fine from the HomeWhiz app etc
Sorry for the lack of information, I will be providing these below

I am using HA Core 2024.12.3. 
The model of my washer dryer is Beko B3YDFT51044W and it works perfectly with the HomeWhiz app.
I used the cloud connection for the HomeWhiz App Integration to HomeAssistant.

**Comment by TechHummel** [comment]
I believe this is the same issue that was fixed with #207. 
I prepared a new release https://github.com/home-assistant-HomeWhiz/home-assistant-HomeWhiz/releases/tag/v0.5.7 - please check if the issue still persists with this version.

**Comment by thomas-ek** [comment]
> I believe this is the same issue that was fixed with #207. 
> I prepared a new release https://github.com/home-assistant-HomeWhiz/home-assistant-HomeWhiz/releases/tag/v0.5.7 - please check if the issue still persists with this version.

It is fixed ! Thank you a lot
